stories/atoms/List.stories.tsx

```diff
@@ -139,3 +140,22 @@ export const _ListOptimizer: Story = {
     },
     render: (args) => <List {...args} />,
 };
+
+export const ListWithSeparators: Story = {
+    name: 'List with separators',
+    args: {
+        isDivided: true,
+        doSelectOnNavigate: true,
+        children: [
+            <ListItem>Next is the separator with no title</ListItem>,
+            <Separator />,
+            <ListItem>Next is the separator with title</ListItem>,
+            <Separator title="Separator title" />,
+            <ListItem>Next is the separator with no title</ListItem>,
+            <Separator />,
+            <ListItem>Next is the separator with title</ListItem>,
+            <Separator title="Separator title" />,
+        ],
+    },
+    render: (args) => <List {...args} />,
+};
```

Comment:
In the other PRs for items we concluded to add examples to Dropdown story, as well as use these components in SelectedItemBadge stories. Can you look into those and replace current usage, or add use-cases with separators?